.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,5 +2,10 @@ Fixes #<issue_number_goes_here>
 
 > It's a good idea to open an issue first for discussion.
 
-- [ ] Tests pass
-- [ ] Appropriate changes to README are included in PR
\ No newline at end of file
+-   [ ] Local tests pass (running `run_desktop_tests.sh`)
```

Comment:
I doesn't necessarily improve test coverage, but it does help advertising this feature to new contributors. Otherwise, they'd always have to wait for workflow feedback. That's why I'd keep it, WDYT?

Reply:
I usually locally test my change only and rely on the workflow for the rest. But yeah we can keep it if it's a useful feature for new contributors.